kqueen/storages/etcd.py

```diff
@@ -398,7 +400,11 @@ def deserialize(cls, serialized, **kwargs):
         object_kwargs = {}
 
         # deserialize toplevel dict and loop fields and deserialize them
-        toplevel = json.loads(serialized)
+        try:
```

Comment:
Are you sure this actually fixes the issue? I think I tried something similar and it did not help